Engine/EditorUI/Panel/PanelResourceDatabase.cpp

```diff
@@ -37,47 +37,18 @@ void PanelResourceDatabase::Render()
 			{
 				++i;
 				ImGui::PushID(pair.first);
-				if (ImGui::Selectable(std::to_string(pair.first).c_str(), selected == i, ImGuiSelectableFlags_SpanAllColumns))
+				if (ImGui::Selectable(std::to_string((unsigned long)pair.first).c_str(), selected == i, ImGuiSelectableFlags_SpanAllColumns))
```

Comment:
why this cast?

Reply:
Because if not, the type `uint32_t` is transformed to an integer when parsing it into a string